pycaw/utils.py

```diff
@@ -120,13 +240,13 @@ def DisplayName(self):
         return s
 
     @DisplayName.setter
-    def DisplayName(self, value):
+    def DisplayName(self, value: str) -> str:
         s = self._ctl.GetDisplayName()
         if s != value:
             self._ctl.SetDisplayName(value, IID_Empty)
```

Comment:
Comments (and typehints) are future lies unless tested.
Are we sure this method returns a string?